### PR TITLE
adding new feature with staging

### LINKED ISSUE #1: Add issue

### PR DESCRIPTION
**Pull request by annacummins**
This pull request fixes #1 

@dom-zero can you check my code please

### CODE REVIEW

File: classify.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 # A list of "random" colors
 COLORS = [
-    "#000000", "#FFFF00", "#1CE6FF", "#FF34FF", "#FF4A46", "#008941", "#006FA6", "#A30059",
+    "#FFFFFF", "#FFFF00", "#1CE6FF", "#FF34FF", "#FF4A46", "#008941", "#006FA6", "#A30059",
```
Comment (dom-zero) [inline classify.py]: sorry don't like this colour - please use something else!

**Review by dom-zero — CHANGES_REQUESTED** [review]

**Comment by annacummins** [comment]
@dom-zero I have updated - please review

**Review by dom-zero — DISMISSED** [review]
all good

File: classify.py [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
 # A list of "random" colors
```
Comment (dom-zero) [inline classify.py]:
```suggestion
# A list of "random" colors  that are nice
```

**Review by dom-zero — APPROVED** [review]

**Pull request merged**